mteb/tasks/Classification/multilingual/IndicSentimentClassification.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import annotations
+
+from typing import Any
+
+import datasets
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks import AbsTaskClassification, MultilingualTask
+
+_LANGUAGES = {
+    "as": ["asm-Beng"],
+    "bd": ["brx-Deva"],
+    "bn": ["ben-Beng"],
+    "gu": ["guj-Gujr"],
+    "hi": ["hin-Deva"],
+    "kn": ["kan-Knda"],
+    "ml": ["mal-Mlym"],
+    "mr": ["mar-Deva"],
+    "or": ["ory-Orya"],
```

Comment:
I found this as `ori` instead: https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes
```suggestion
    "or": ["ori-Orya"],
```

Reply:
`iso_639_3_to_language.json` in the repository uses `ory` as the language code for Odia.

https://github.com/embeddings-benchmark/mteb/blob/main/mteb/abstasks/iso_639_3_to_language.json#L5089